picard/ui/mainwindow.py

```diff
@@ -397,6 +397,9 @@ def create_actions(self):
         self.cd_lookup_action.triggered.connect(self.tagger.lookup_cd)
 
         self.analyze_action = QtGui.QAction(icontheme.lookup('picard-analyze'), _(u"&Scan"), self)
+        tip = _(u"Use AcoustID audio fingerprint to identify the files by the actual music, even if they have no metadata")
+        self.analyze_action.setToolTip(tip)
```

Comment:
The other actions only have a statusTip. To keep consistency with them, please remove this line.